src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Lottery lottery = new Lottery(colorSupplier);
+
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+
+        System.out.println("Ball 1: " + ball1);
+        System.out.println("Ball 2: " + ball2);
+        System.out.println("Ball 3: " + ball3);
```

Comment:
make sure to avoid [checklist ](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-for-loop-for-creating-several-objects-of-the-same-class) mistakes